### PR TITLE
Update how_to_train.md

### PR DESCRIPTION
**Pull request by halfrot**
fix description about `min_new_tokens`

### CODE REVIEW

**Review by younesbelkada — APPROVED** [review]
I think that looks correct, wdyt @lvwerra @vwxyzjn ?

File: docs/source/how_to_train.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ To address this issue, we add a penalty to the reward function based on the KL d
...
 
 - **top-k sampling**: the model can smooth out the probability distribution causing the top-k tokens having a smaller probability than those of the reference model but they still are selected
-- **min_length**: this ignores the EOS token until `min_length` is reached. thus the model can assign a very high log prob to the EOS token and very low prob to all others until min_length is reached
+- **min_length**: this ignores the EOS token until `min_length` is reached. thus the model can assign a very low log prob to the EOS token and very high probs to all others until min_length is reached
 - **batched generation**: finished sequences in a batch are padded until all generations are finished. The model can learn to assign very low probabilities to the padding tokens unless they are properly masked or removed.
```
Comment (vwxyzjn) [inline docs/source/how_to_train.md:33]: should be a comma instead a period.
```suggestion
- **min_length**: this ignores the EOS token until `min_length` is reached, thus the model can assign a very low log prob to the EOS token and very high probs to all others until min_length is reached
```

Comment (vwxyzjn) [inline docs/source/how_to_train.md:33]:
```suggestion
- **min_length**: this ignores the EOS token until `min_length` is reached, thus the model can assign a very low log prob to the EOS token and very high probs to all others until min_length is reached
```suggestion
- **min_length**: this ignores the EOS token until `min_length` is reached, thus the model can assign a very low log prob to the EOS token and very high probs to all others until min_length is reached
```

Comment (vwxyzjn) [reply docs/source/how_to_train.md:33]: Not sure why the commit suggestion was weird 🤦‍♂️

Comment (halfrot) [reply docs/source/how_to_train.md]: lol, I see

**Review by younesbelkada — APPROVED** [review]
Thanks again !

**Pull request merged**